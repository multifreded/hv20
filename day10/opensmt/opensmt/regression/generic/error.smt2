(assert foo)
(set-logic AB_CD)
