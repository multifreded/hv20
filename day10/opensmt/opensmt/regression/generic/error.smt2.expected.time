Command terminated by signal 11
real 0.19
user 0.00
sys 0.00
