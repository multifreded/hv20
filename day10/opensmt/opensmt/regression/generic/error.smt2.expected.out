(error "Illegal command before set-logic: assert")

(error "unknown logic AB_CD")

